pgjdbc/src/test/java/org/postgresql/test/TestUtil.java

```diff
@@ -268,22 +266,19 @@ public static Properties loadPropertyFiles(String... names) {
     return p;
   }
 
-  public static void initDriver() {
-    synchronized (TestUtil.class) {
-      if (initialized) {
-        return;
-      }
+  public static synchronized void initDriver() {
+    if (Driver.isRegistered()) {
+      return;
+    }
 
-      try {
-        Driver.register();
-      } catch (Exception e) { }
+    try {
+      Driver.register();
+    } catch (Exception e) { }
```

Comment:
Let's remove try-catch block. This test utility is used in test suite and it makes sense to know about all issues. Eating exception isn't a good approach here.